python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -532,7 +532,12 @@ def __init__(self, model: PyFuncModel,
 
         self._model = model
         self._classes = classes
-        self._label_dict = {label: (i+1) for i, label in enumerate(classes)}
+        try:
+            assert type(self._classes[0]) == str
```

Comment:
asserts are only used for tests - in package code we throw exceptions instead when validating code